tgui/packages/tgui/components/NanoMap.js

```diff
@@ -22,12 +21,12 @@ export class NanoMap extends Component {
     super(props);
 
     // Auto center based on window size
-    const Xcenter = 0;
+    const Xcenter = window.innerWidth / 2 - 256;
```

Comment:
ЗАЧЕМ, я специально 0 ставил чтобы эта залупа центрировалась
```suggestion
    const Xcenter = window.innerWidth / 2 - 256;
```

Reply:
А я не зал, тупо ориг файл оффов скопировал